apps/condo/domains/notification/access/PushToken.js

```diff
@@ -0,0 +1,55 @@
+// @ts-nocheck
+/**
+ * Generated by `createschema notification.PushToken PushToken`
+ */
+
+const { throwAuthenticationError } = require('@condo/domains/common/utils/apolloErrorFormatter')
+
+async function canReadPushTokens ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin) return {}
+
+    return {
+        context: {
+            OR: [
+                {
+                    owner: { id: user.id },
+                },
+                // may be extended in future
+            ],
+        },
+    }
+}
+
+async function canManagePushTokens ({ authentication: { item: owner }, originalInput, operation }) {
```

Comment:
use user for arguments like in other access functions. Try not to change the common style!

Reply:
fixed